src/Slider.jsx

```diff
@@ -516,11 +518,14 @@ class Rheostat extends React.Component {
   }
 
   positionPercent(x, y, sliderBox) {
-    const { orientation } = this.props;
+    const { orientation, direction } = this.props;
     if (orientation === VERTICAL) {
       return ((y - sliderBox.top) / sliderBox.height) * PERCENT_FULL;
     }
-    return ((x - sliderBox.left) / sliderBox.width) * PERCENT_FULL;
+    if (direction === DIRECTIONS.RTL) {
+      return 100 - (((x - sliderBox.left) / sliderBox.width) * PERCENT_FULL);
```

Comment:
Can we pull the `((x - sliderBox.left) / sliderBox.width) * PERCENT_FULL` out into its own variable?